Components/Images/images.css

```diff
@@ -0,0 +1,29 @@
+@import url("/CSS/main.css");
+/* * {
+  outline: 1px solid red;
+} */
+.image-wrapper {
+  display: flex;
```

Comment:
You can try making `display : flex` a utility class and reuse it whenever possible